java-runtime/src/main/java/io/spine/validate/Validate.java

```diff
@@ -208,7 +210,9 @@ public static <M extends Message> void checkValidChange(M previous, M current) {
      *         the type of the message
      * @return a set of constraint violations, if the transaction is invalid,
      *         an empty set otherwise
+     * @deprecated the {@code set_once} constraint is enforced within the message builder now.
```

Comment:
Again, as per my comment above, we don't give the user instructions on what he should do about the deprecation. Please rephrase.